app/views/children/index.html.erb

```diff
@@ -30,72 +30,62 @@
             <h3 class="card-title">Filters</h3>
           </div>
           <div class="card-body">
-            <%= form_tag(children_path, method: :get) do |f| %>
+            <%= form_for_filterrific @filterrific, remote: true do |f| %>
               <div class="row">
-                <% if @family.present? %>
-                  <div class="form-group col-lg-4 col-md-4 col-sm-6 col-xs-12">
-                    <%= label_tag "by Guardian Name", "Guardian Name" %>
-                    <%= collection_select(:filters, :from_family, @family || {}, :id, :guardian_display_name, {include_blank: true, selected: @selected_family}, class: "form-control") %>
-                  </div>
-                <% end %>
-                <% if @children.present? %>
-                  <div class="form-group col-lg-4 col-md-4 col-sm-6 col-xs-12">
-                    <%= label_tag "by Children First Name", "Child's First Name" %>
-                    <%= collection_select(:filters, :from_children, @children || {}, :id, :first_name, {include_blank: true, selected: @selected_children_first_name}, class: "form-control") %>
-                  </div>
-                <% end %>
+                <div class="col-2">
+                  <%= f.label :search_names, "Search By Child Name" %>
+                  <%= f.text_field(
+                          :search_names,
+                          class: 'filterrific-periodically-observed form-control'
+                      ) %>
+                </div>
+                <div class="col-2">
+                  <%= f.label :search_families, "Search By Guardian Name" %>
+                  <%= f.text_field(
+                          :search_families,
+                          class: 'filterrific-periodically-observed form-control'
+                      ) %>
+                </div>
+                <div class="col-2">
+                  <%= f.label :search_comments, "Search By Active Children" %>
```

Comment:
```suggestion
                  <%= f.label :search_comments, "Show Active Only?" %>
```
A minor suggestion here. I read "Search By Active Children" to sort of implying that the Child is active. I think making 'Show Active Only?" might be a bit clearer though I can see the original working too :)

Reply:
Changed!